examples/contextual_mab.py

```diff
@@ -150,3 +150,31 @@
 
 # Updating of the model with new arm
 clusters.add_arm(6)
+
+###############################################################
+# LSH Approximate Neighborhood Policy with UCB1 Learning Policy
+###############################################################
+
+# Radius contextual policy with radius equals to 5 and ucb1 learning with alpha 1.25
+ann = MAB(arms=ads,
+          learning_policy=LearningPolicy.UCB1(alpha=1.25),
+          neighborhood_policy=NeighborhoodPolicy.LSHNearest(n_dimensions=5, n_tables=5))
+
+# Learn from previous ads shown and revenues generated
+ann.fit(decisions=train_df['ad'], rewards=train_df['revenues'], contexts=train)
```

Comment:
consistency question: in other examples what do we call these objects, "mab"? or say "greedy" or "radius" etc.

Reply:
For the examples we name the object greedy, radius, etc. I'll change it to lshnearest